entrypoint.sh

```diff
@@ -10,19 +10,19 @@ source "$PRJ_ROOT"/bin/aws_minikube.sh
 source "$PRJ_ROOT"/bin/argocd.sh
 
 
-# Variables passed by Github Workflow to Action
+# Mandatory variables passed to container by config
 readonly CLUSTER_CONFIG_PATH=$1
 readonly CLOUD_USER=$2
 readonly CLOUD_PASS=$3
-readonly GIT_REPO_ROOT=$GITHUB_WORKSPACE
-# TODO: Create function to select git repo root from ENV for GitLab and BitBucket
 
+# Detect Git hosting and set GIT_REPO_NAME, GIT_REPO_ROOT variables
```

Comment:
```suggestion
# Detect Git hosting and set GIT_PROVIDER, GIT_REPO_NAME, GIT_REPO_ROOT constants
```

Reply:
> 

At least at  ArgoCD  argocd::deploy_apps, or you think we could ovveride this with relative path?